[2025-02-06T17:19:21.191+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T17:19:21.210+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_transformation_storage_pipeline.insert_selected_data manual__2025-02-06T17:19:15.004944+00:00 [queued]>
[2025-02-06T17:19:21.225+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_transformation_storage_pipeline.insert_selected_data manual__2025-02-06T17:19:15.004944+00:00 [queued]>
[2025-02-06T17:19:21.229+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T17:19:21.247+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): insert_selected_data> on 2025-02-06 17:19:15.004944+00:00
[2025-02-06T17:19:21.255+0000] {standard_task_runner.py:72} INFO - Started process 9055 to run task
[2025-02-06T17:19:21.259+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_transformation_storage_pipeline', 'insert_selected_data', 'manual__2025-02-06T17:19:15.004944+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/data_transformation_storage_pipeline.py', '--cfg-path', '/tmp/tmpl7e227be']
[2025-02-06T17:19:21.263+0000] {standard_task_runner.py:105} INFO - Job 62: Subtask insert_selected_data
[2025-02-06T17:19:21.547+0000] {task_command.py:467} INFO - Running <TaskInstance: data_transformation_storage_pipeline.insert_selected_data manual__2025-02-06T17:19:15.004944+00:00 [running]> on host c3d37860668a
[2025-02-06T17:19:21.679+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_transformation_storage_pipeline' AIRFLOW_CTX_TASK_ID='insert_selected_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T17:19:15.004944+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-06T17:19:15.004944+00:00'
[2025-02-06T17:19:21.683+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T17:19:21.708+0000] {warnings.py:110} WARNING - /opt/***/dags/data_transformation_storage_pipeline.py:56: FutureWarning: Passing literal json to 'read_json' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df = pd.read_json(json_data)

[2025-02-06T17:19:21.734+0000] {warnings.py:110} WARNING - /opt/***/dags/data_transformation_storage_pipeline.py:60: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  df = df.applymap(lambda x: x.strip() if isinstance(x, str) else x)

[2025-02-06T17:19:21.744+0000] {baseoperator.py:421} WARNING - PostgresOperator.execute cannot be called outside TaskInstance!
[2025-02-06T17:19:21.746+0000] {sql.py:278} INFO - Executing: 
            INSERT INTO car_data (brand, model, body_style, seat, price)
            VALUES (?, ?, ?, ?, ?)
        
[2025-02-06T17:19:21.763+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-02-06T17:19:21.774+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-02-06T17:19:21.778+0000] {sql.py:553} INFO - Running statement: 
            INSERT INTO car_data (brand, model, body_style, seat, price)
            VALUES (?, ?, ?, ?, ?)
        , parameters: ('Tesla', 'Model 3 Long Range Dual Motor', 'Sedan', 5, 55480)
[2025-02-06T17:19:21.780+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/data_transformation_storage_pipeline.py", line 77, in insert_selected_data
    postgres_operator.execute(context=None)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/operators/sql.py", line 284, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/hooks/sql.py", line 498, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/hooks/sql.py", line 556, in _run_command
    cur.execute(sql_statement, parameters)
psycopg2.errors.SyntaxError: syntax error at or near ","
LINE 3:             VALUES (?, ?, ?, ?, ?)
                             ^

[2025-02-06T17:19:21.796+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=data_transformation_storage_pipeline, task_id=insert_selected_data, run_id=manual__2025-02-06T17:19:15.004944+00:00, execution_date=20250206T171915, start_date=20250206T171921, end_date=20250206T171921
[2025-02-06T17:19:21.819+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T17:19:21.822+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 62 for task insert_selected_data (syntax error at or near ","
LINE 3:             VALUES (?, ?, ?, ?, ?)
                             ^
; 9055)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/data_transformation_storage_pipeline.py", line 77, in insert_selected_data
    postgres_operator.execute(context=None)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/operators/sql.py", line 284, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/hooks/sql.py", line 498, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/hooks/sql.py", line 556, in _run_command
    cur.execute(sql_statement, parameters)
psycopg2.errors.SyntaxError: syntax error at or near ","
LINE 3:             VALUES (?, ?, ?, ?, ?)
                             ^

[2025-02-06T17:19:21.877+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-02-06T17:19:22.091+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-06T17:19:22.101+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
